### PR TITLE
Update long-term-support-releases.md

### PR DESCRIPTION
**Pull request by carolwkc**
## Description
Update LTS information for Pump, MDCB, TIB, and Operator

## Types of changes

- [ ] Fixing typo (please merge to production)
- [ ] Documenting a new feature (please merge to production)
- [ ] Documentation for future release (please do not merge to production)
- [ ] Something else (please add if needs merging to production or not)

## Checklist

- [ ] I have [reviewed the guidelines](../CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](../CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I have added a preview link.

### CODE REVIEW

File: tyk-docs/content/frequently-asked-questions/long-term-support-releases.md [hunk 1]
```diff
@@ -69,18 +69,14 @@ In June, version 5 becomes the recommended version; we will release the next maj
...
-
-To help assure backward compatibility we ensure that the each version of Pump we release works with the gateway and dashboard versions which is under long term support at that time. For example, we'd ensure Pump 1.7 is compatible with release 4 of Tyk gateway and Tyk Manager (dashboard), and this increments with the long term support model.
+Minor updates to Tyk Pump, Tyk Identity Broker (TIB),  MDCB, and Operator are deployable quickly with low risk and no breaking changes, Tyk will support the latest major.minor version (e.g. Pump 1.7) until the next major.minor version (1.8) is released. Then we'd support that version.
 
-If we need to patch a major Pump version , we would number that as a minor version (1.7.1).
+To help assure backward compatibility we ensure that each version of Pump and other components we release works with the gateway and dashboard versions which is under long term support at that time. For example, we'd ensure Pump 1.7 is compatible with release 4 of Tyk gateway and Tyk Manager (dashboard), and this increments with the long term support model.
```
Comment (letzya) [inline tyk-docs/content/frequently-asked-questions/long-term-support-releases.md]:
```suggestion
To help assure backward compatibility we ensure that each version of Pump and other components we release works with the gateway and dashboard versions which are under LTS (long-term support) at that time. For example, we'd ensure Pump v1.7 is compatible with release v4 of Tyk gateway and Tyk Dashboard, and this increments with the LTS model.
```

Comment (letzya) [reply tyk-docs/content/frequently-asked-questions/long-term-support-releases.md]: @caroltyk what do you think

**Review by letzya — CHANGES_REQUESTED** [review]
asked for changes

**Comment by carolwkc** [comment]
> asked for changes

Updated

**Review by letzya — APPROVED** [review]

**Pull request merged**